pymc_marketing/mmm/utils.py

```diff
@@ -326,3 +273,24 @@ def apply_sklearn_transformer_across_date(
     data.attrs = attrs
 
     return data
+
+
+def sigmoid_saturation(
+    x: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    alpha: Union[float, np.ndarray, npt.NDArray[np.float64]],
+    lam: Union[float, np.ndarray, npt.NDArray[np.float64]],
+) -> Union[float, Any]:
+    """
+    Parameters
+    ----------
+    - alpha
+        α (alpha): Represent the Asymptotic Maximum or Ceiling Value.
+    - lam
+        λ (lambda): affects how quickly the function approaches its upper and lower asymptotes. A higher value of
+        lam makes the curve steeper, while a lower value makes it more gradual.
```

Comment:
Can you please prettify the docs a bit? It seems we have many "-" , see https://pymc-marketing--541.org.readthedocs.build/en/541/api/generated/pymc_marketing.mmm.utils.sigmoid_saturation.html :)